packages/sx05re/emuelec/bin/emuelec-utils

```diff
@@ -263,7 +269,7 @@ function resolutions() {
 	fi
 
 	if [[ -f "$FILEPATH" ]]; then
-	  SCREEN_SUPPORTED_RESOLUTIONS=`/usr/bin/cat '$FILEPATH'`
+	  SCREEN_SUPPORTED_RESOLUTIONS=`/usr/bin/cat "$FILEPATH"`
```

Comment:
this is a minor fix from a different issue. I noticed that the call was throwing an error.